docker/entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+composer install
```

Comment:
not needed for the same reason mentioned before.

Reply:
this should be done inside the image build, to package it with all its dependencies.

i suggest using the builder pattern.

Reply:
@calvinbui I agree, the dependencies would be best installed within the image itself.
As I explained the the commit message though, this is not currently possible:
>Note that here, running 'composer install' is added to the entrypoint
and not the Dockerfile because it seems to require the SQLite database
file to exist already at that time

Since installing from the Dockerfile requires more changes, I went for installing at runtime as an intermediate step to already have _something_ that can be used to run the app.